### PR TITLE
Remove flipY and premultiplyAlpha from the test

### PR DESCRIPTION
**Pull request by xidachen**
@zhenyao @toji @kenrussell Please review, both parameters are now removed from the test

### CODE REVIEW

File: sdk/tests/js/tests/tex-image-and-sub-image-3d-with-image-bitmap.js [hunk 1]
```diff
@@ -139,12 +130,11 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
 
     function runTestOnBindingTarget(bindingTarget, program) {
         var cases = [
-            { flipY: false, premultiplyAlpha: false },
+            { },
```
Comment (zhenyao) [inline sdk/tests/js/tests/tex-image-and-sub-image-3d-with-image-bitmap.js]: You can get rid of this cases.

File: sdk/tests/js/tests/tex-image-and-sub-image-3d-with-image-bitmap.js [hunk 2]
```diff
@@ -73,10 +73,9 @@ function generateTest(internalFormat, pixelFormat, pixelType, prologue, resource
         });
     }
 
-    function runOneIteration(useTexSubImage2D, flipY, premultiplyAlpha, bindingTarget, program)
+    function runOneIteration(useTexSubImage2D, bindingTarget, program)
```
Comment (zhenyao) [inline sdk/tests/js/tests/tex-image-and-sub-image-3d-with-image-bitmap.js]: You can also get rid of useTexSubImage2D as it is always true and this arg is not used in the function at all.

**Comment by xidachen** [comment]
Thanks for catching it. The cases parameter has been removed.

**Comment by zhenyao** [comment]
lgtm

**Pull request merged**